c/src/edc.c

```diff
@@ -74,7 +74,9 @@ static const u16 crc16tab[256] = {
 u16 crc16_ccitt(const u8 *buf, u32 len, u16 crc)
 {
   for (u32 i = 0; i < len; i++)
+  {
```

Comment:
I'm a little surprised this curly brace wasn't put onto the above line. Do we not run `clang-format` on this code?

Reply:
No clang-format is completely disabled in this repo, I have no idea why. After this is merged I can go back and add it in, probably won't do it right now since I suspect there will be thousands of changes.

Reply:
I guess the autogenerated code might not be formatted correctly, or is difficult to handle all the cases that the formatter could complain about. Might not be worth your time digging into it

Reply:
Yeah I had a quick go turning it on here https://github.com/swift-nav/libsbp/pull/911 it's not pretty. At a minimum I think generated code would have to be omitted.